server/game/cards/Deluxe/LawOfDomination.js

```diff
@@ -11,12 +11,12 @@ class LawOfDomination extends Card {
                 theirs: {
                     player: 'opponent',
                     controller: 'opponent',
-                    CardType: BattlefieldTypes,
+                    cardType: BattlefieldTypes,
```

Comment:
these let you target any type of card. like attack their unit with your ready spell. which caused a server error